pydarn/plotting/rtp.py

```diff
@@ -506,7 +489,226 @@ def plot_time_series(cls, dmap_data: List[dict], *args,
         ax.xaxis.set_major_formatter(dates.DateFormatter(date_fmt))
         ax.xaxis.set_minor_locator(dates.HourLocator())
         ax.margins(x=0)
-        return lines
+        ax.tick_params(axis='y', which='minor')
+        return lines, x, y
+
+    @classmethod
+    def plot_summary(cls, dmap_data: List[dict], *args, beam_num: int = 0,
```

Comment:
Question: why is `*args` placed where it is?

Reply:
Funny store, a while back I was having issues calling a function with `args` and `kwargs` and thought it had to be positioned before the variable names, ends up not. Looks like it was one of those artifacts that happen when debugging but never went back to normal. 

I actually think we don't need args (I don't use the args) just the kwargs for the matplotlib plotting. So I will remove them :)